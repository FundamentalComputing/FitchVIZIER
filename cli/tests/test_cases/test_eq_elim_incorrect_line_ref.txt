1 | a=b
  | ---
2 | a=a   =Intro
3 | b=a   =Elim:2,4