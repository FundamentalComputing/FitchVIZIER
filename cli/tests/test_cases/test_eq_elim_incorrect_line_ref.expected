Line 3: line 4 is being referenced in the justification, but that line does not exist.